lib/puppet_forge/v3/metadata.rb

```diff
@@ -91,13 +94,17 @@ def dependencies
       def to_hash
         @data
       end
-      alias :to_data_hash :to_hash
+      alias to_data_hash to_hash
 
-      def to_json
+      def to_json(*_args)
         data = @data.dup.merge('dependencies' => dependencies)
 
         contents = data.keys.map do |k|
```

Comment:
I think rubocop-performance should catch this and iterate on the pair
```suggestion
        contents = data.map do |k, v|
```